tests/integration/pages/experiment_timeline_and_population.py

```diff
@@ -126,30 +125,6 @@ def firefox_max_version(self, version=None):
             if item.get_attribute("value") == version:
                 item.click()
 
-    @property
-    def locale(self):
-        element = self.find_element(*self._locale_btn_locator)
-        # scroll to element
-        self.selenium.execute_script("arguments[0].scrollIntoView(true);", element)
-        self.selenium.execute_script("arguments[0].click();", element)
-        locales = self.selenium.find_elements(By.CSS_SELECTOR, "div.show:nth-child(3) a")
-        for item in locales:
-            if item.get_attribute("aria-selected") == "true":
-                locale_text = item.find_element(By.CSS_SELECTOR, "span.text").text
-                self.selenium.execute_script("arguments[0].click();", element)
-                return locale_text
-
-    @locale.setter
-    def locale(self, locale=None):
-        element = self.find_element(*self._locale_btn_locator)
-        self.selenium.execute_script("arguments[0].click();", element)
-        locales = self.selenium.find_elements(By.CSS_SELECTOR, "div.show:nth-child(3) a")
-        for item in locales:
-            if locale in item.find_element(By.CSS_SELECTOR, "span.text").text:
-                self.selenium.execute_script("arguments[0].click();", item)
-                break
-        self.selenium.execute_script("arguments[0].click();", element)
-
```

Comment:
In this PR I've eventually landed on taking out the locales test. I couldn't figure out how to write a selenium test to grab the right elements. I think it's made much more complicated by this the element being made by a third party lib, "react-select".. there are many divs this library creates for the element with weird autogenerated class names... essentially there's a lot of divs that I can't really control. I can't add helper classes to things to make the grabbing of elements easier. I'm wondering... do we need to test the locales field box, especially when we aren't testing every field on this page with integration tests? I test the react-select element function in two other places in the front-end tests. If we really want to integration test the selects on this page, perhaps Ben can help me on this, since he's the expert here.

Reply:
Yeah I think it's okay to drop these, @jrbenny35 is adding a bunch of new tests that will cover filling out all the forms and walking through the rest of the flow, and as those land I think we'll be able to drop all these field specific tests, so ya this is fine.